public/app/plugins/datasource/loki/LanguageProvider.ts

```diff
@@ -260,4 +284,13 @@ export default class LokiLanguageProvider extends LanguageProvider {
       hasLogfmt,
     };
   }
+
+  /**
+   * Get the default time range
+   *
+   * @returns {TimeRange} The default time range
+   */
+  getDefaultTimeRange(): TimeRange {
```

Comment:
This looks like it should be private.

Reply:
Updated in https://github.com/grafana/grafana/pull/78450/commits/3b0f7f30c10194cea5154112dab37a19c86d7ba4